brion/plugin/spikeReportASCII.cpp

```diff
@@ -48,7 +46,8 @@ Spikes SpikeReportASCII::read( const float )
     Spikes spikes;
     auto start = _lastReadPosition;
     _lastReadPosition = _spikes.end();
-    _currentTime = UNDEFINED_TIMESTAMP;
+    _currentTime = _spikes.rbegin()->first +
+                   std::numeric_limits< float >::epsilon();
```

Comment:
The spec doesn't say anything about the value of getCurrentTime() when state is ended, it's undefined. For that reason, your code shouldn't rely on any specific value. My gut tells me this change is wrong if you don't constraint the specification, but referring to deltas there also seems wrong. Do you really need this change to make the incremental converter work?